app/controllers/templates_controller.rb

```diff
@@ -87,11 +87,18 @@ def preview
     else
       @template = resource_class.new(params[type_name_plural])
```

Comment:
I was trying to see if we could send it as `template_kind_id` and feed it here, but I think `type_name_plural` is not working for some time :facepalm: